Tools/clinic/libclinic/formatting.py

```diff
@@ -59,11 +60,7 @@ def wrapped_c_string_literal(
     return initial_indent * " " + c_repr(separator.join(wrapped))
 
 
-def _add_prefix_and_suffix(
-    text: str,
-    prefix: str = "",
-    suffix: str = ""
-) -> str:
+def _add_prefix_and_suffix(text: str, prefix: str = "", suffix: str = "") -> str:
```

Comment:
It might be nice to make `prefix` and `suffix` keyword-only here, so that it's impossible to accidentally pass a value to `prefix` when we meant to pass it to `suffix`

```suggestion
def _add_prefix_and_suffix(text: str, *, prefix: str = "", suffix: str = "") -> str:
```

Reply:
Yep, I agree.